spec/requests/posts_request_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET #index' do
+    it 'deploys index action template' do
+      get '/users/1/posts'
+      expect(response).to render_template(:post_list)
+    end
+
+    it 'is a success' do
+      get '/users/1/posts'
+      expect(response).to have_http_status(:ok)
+    end
+  end
+
+  describe 'GET #show' do
+    it 'deploys show action template' do
+      get '/users/1/posts/1'
+      expect(response).to render_template(:post_details)
+    end
+
+    it 'is a success' do
+      get '/users/1/posts/1'
+      expect(response).to have_http_status(:ok)
+    end
+  end
+end
```

Comment:
### OPTIONAL
_I wrote it as an optional but this is a requirement that was not mentioned in the last review. Please fix it._
- [ ] Spec file should include tests that ensure the response body includes the correct placeholder text.
 Example : `expect(response.body).to include("Welcome to user index page")`